app/src/main/java/campus/tech/kakao/map/model/Location.kt

```diff
@@ -0,0 +1,13 @@
+package campus.tech.kakao.map.model
+
+data class Location(
+    val title: String,
+    val address: String,
+    val category: String
+){
+    companion object {
+        fun toLocation(locationDto: LocationDto): Location {
+            return Location(locationDto.place_name, locationDto.address_name, locationDto.category_group_name)
+        }
```

Comment:
확장함수로 제공된다면 조금 더 좋은 형태일 것 같네요!
```suggestion
        fun LocationDto.toLocation(): Location {
            return Location(place_name, address_name, category_group_name)
        }
```